backend/pkg/api/handlers/auth.go

```diff
@@ -253,6 +289,146 @@ func (h *HandlerService) InternalPostLogout(w http.ResponseWriter, r *http.Reque
 	returnOk(w, nil)
 }
 
+func (h *HandlerService) InternalDeleteUser(w http.ResponseWriter, r *http.Request) {
+	user, err := h.getUserBySession(r)
+	if err != nil {
+		handleErr(w, err)
+		return
+	}
+
+	// TODO allow if user has any subsciptions etc?
+	err = h.dai.RemoveUser(user.Id)
+	if err != nil {
+		handleErr(w, err)
+		return
+	}
+
+	returnNoContent(w)
+}
+
+func (h *HandlerService) InternalPutUserEmail(w http.ResponseWriter, r *http.Request) {
```

Comment:
Btw, if you add the 409 response codes this should be documented in the API spec 😅